src/views/CashlinkManage.vue

```diff
@@ -17,38 +17,34 @@
-                    </svg>
-                </a>
+                <template v-else>
+                    <a class="nq-button-s social-share telegram" :href="telegram" target="_blank">
+                        <svg width="29" height="25" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
+                            <path opacity=".6" d="M28.537.528c-.103-.308-.224-.397-.416-.472-.42-.164-1.124.084-1.124.084S1.943 9.496.516 10.534c-.308.225-.416.351-.467.505-.247.74.523 1.062.523 1.062l6.458 2.185s.023.005.056.005l3.024 8.7s.275.585.61.777c.01.01.024.014.033.023l.028.014c.038.019.08.028.131.028.005 0 .01.005.014.005h.005c.252 0 .658-.22 1.311-.903a56.694 56.694 0 013.392-3.227c2.221 1.595 4.61 3.354 5.642 4.275.518.463.952.538 1.306.524.98-.038 1.251-1.155 1.251-1.155s4.573-19.09 4.722-21.65c.014-.252.038-.411.038-.584.004-.239-.019-.477-.056-.59zM8.332 13.758l.004-.132c3.463-2.268 13.887-9.089 14.568-9.35.122-.038.21.004.187.093-.308 1.123-11.866 11.797-11.866 11.797s-.042.056-.07.122l-.014-.01-.397 4.398-2.412-6.919z"/>
+                        </svg>
+                    </a>
+                    <a class="nq-button-s social-share" :href="mail">
+                        <svg width="28" height="24" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
+                            <g opacity=".6">
+                                <path d="M13.905 11.364c.133.134.267.134.4 0l12.836-8.556c.267-.134.4-.401.267-.669A2.732 2.732 0 0024.734 0H2.674A2.732 2.732 0 000 2.14c0 .267.134.534.401.801l13.504 8.423z"/>
+                                <path d="M15.375 14.44c-.802.534-1.738.534-2.54 0L1.07 6.952c-.402-.133-.803 0-.936.268C0 7.353 0 7.487 0 7.62v13.103a2.682 2.682 0 002.674 2.674h22.06a2.682 2.682 0 002.674-2.674V7.621c0-.401-.267-.669-.668-.669-.134 0-.268 0-.401.134L15.375 14.44z"/>
+                            </g>
+                        </svg>
+                    </a>
+                    <a class="nq-button-s social-share" :href="whatsapp" target="_blank">
+                        <svg width="28" height="28" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
+                            <path opacity=".6" d="M14 0C6.3 0 0 6.235 0 14c0 2.823.817 5.647 2.45 8L.817 26.706c-.117.235 0 .47.116.588.117.118.35.235.584.118l5.016-1.53C13.067 30 21.7 28 25.783 21.412c1.4-2.235 2.1-4.824 2.1-7.412C28 6.235 21.7 0 14 0zm8.4 19.647c-.817 1.53-2.333 2.47-4.083 2.47-1.4-.117-2.8-.588-3.967-1.176-2.8-1.294-5.25-3.412-7-6.117-2.217-2.942-2.333-5.765-.233-8.118.583-.47 1.4-.706 2.216-.588.584.117 1.167.47 1.4 1.058l.467 1.06c.35.823.7 1.646.7 1.764.233.353.233.823 0 1.177-.35.588-.7 1.176-1.167 1.646a10.68 10.68 0 002.1 2.471c.934.824 1.984 1.53 3.15 1.883.35-.471.934-1.177 1.167-1.53a1.175 1.175 0 011.517-.47c.466.117 2.916 1.411 2.916 1.411.35.118.584.353.817.588.467.942.467 1.765 0 2.471z"/>
+                        </svg>
```

Comment:
Once decimal can be shaved off all coordinates in this SVG without any visual change:

```html
<svg width="28" height="28" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
    <path opacity=".6" d="M14 0A14 14 0 000 14c0 2.82.82 5.65 2.45 8L.82 26.7c-.12.24 0 .48.11.6.12.11.35.23.59.11l5.01-1.53a13.91 13.91 0 0019.25-4.47c1.4-2.23 2.1-4.82 2.1-7.41C28 6.24 21.7 0 14 0zm8.4 19.65a4.57 4.57 0 01-4.08 2.47c-1.4-.12-2.8-.6-3.97-1.18a16.76 16.76 0 01-7-6.12c-2.22-2.94-2.33-5.76-.23-8.11a2.9 2.9 0 012.21-.6c.59.12 1.17.48 1.4 1.07l.47 1.06c.35.82.7 1.64.7 1.76.23.35.23.82 0 1.18-.35.58-.7 1.17-1.17 1.64a10.68 10.68 0 002.1 2.47 8.68 8.68 0 003.15 1.89c.35-.47.94-1.18 1.17-1.53a1.18 1.18 0 011.52-.47c.46.11 2.91 1.4 2.91 1.4.35.13.59.36.82.6.47.94.47 1.76 0 2.47z"/>
</svg>
```